### PR TITLE
fix: `debug` rpc should return a list of active debug categories, not all of them

### PR DESCRIPTION
**Pull request by UdjinM6**
## Issue being fixed or feature implemented
This restores previous behaviour which was changed/broken here https://github.com/dashpay/dash/commit/e554d3a02e58a298c9f2b10a8bee443cbfc8ed54#diff-0ba691cbdd97c095286e9373ed8d5be87d559234440487956326965e16cbb421R75

## What was done?
Fix `debug` rpc results

## How Has This Been Tested?
Run rpc, check results

## Breaking Changes
n/a

## Checklist:
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated relevant unit/integration/functional/e2e tests
- [ ] I have made corresponding changes to the documentation
- [x] I have assigned this pull request to a milestone _(for repository code-owners and collaborators only)_

### CODE REVIEW

File: src/rpc/misc.cpp [hunk 1]
```diff
@@ -71,7 +71,7 @@ static UniValue debug(const JSONRPCRequest& request)
         }
     }
 
-    return "Debug mode: " + LogInstance().LogCategoriesString();
+    return "Debug mode: " + LogInstance().LogCategoriesString(/*enabled_only*/ true);
```
Comment (PastaPastaPasta) [inline src/rpc/misc.cpp:74]:
```suggestion
    return "Debug mode: " + LogInstance().LogCategoriesString(/*enabled_only=*/true);
```

**Review by PastaPastaPasta — APPROVED** [review]
nit, should use the proper format for argument name comments otherwise utACK for squash

**Review by ogabrielides — APPROVED** [review]
LGTM

**Review by PastaPastaPasta — APPROVED** [review]
utACK for squash merge

**Pull request merged**